docs/introduction.md

```diff
@@ -42,7 +44,34 @@ The final version of the previous execution state remains available through a le
 
 ## Flow
 
-Flow was designed to provide a blockchain that can scale while preserving composability in a single blockchain state. This is achieved by introducing several [node roles](#nodes), each with their own trade-off for the [scalability trilemma](https://aakash-111.medium.com/the-scalability-trilemma-in-blockchain-75fb57f646df).
+Most current blockchains are built as a homogenous system comprised of *full nodes*.
+Each full node is expected to collect and choose the transactions to be included in the next block, execute the block, come to a consensus over the output of the block with other full nodes, and finally sign the block and append it to the chain.
+
+Existing blockchain architectures have well documented throughput limitations. There are two main approaches to combat this:
+
+1. Layer-2 networks
+2. Sharding
```

Comment:
That's not really much of a different, to be honest. Layer-2 networks also create a certain sub-state of the main consensus state, where more transactions happen. I guess the only difference between sharding and layer-2 solutions is whether the main state is divided into several states.